src/main/resources/data/minecolonies/tags/entity_types/mob_attack_blacklist.json

```diff
@@ -0,0 +1,6 @@
+{
+  "replace": false,
+  "values": [
+    "minecraft:slime"
+  ]
```

Comment:
Looks mostly fine, here I'd remove the slime, and add Llama and Enderman (so that you can remove them in the code above.

Reply:
that actually makes way more sense than using slime as a placeholder... I don't know why that didn't translate in my head.